server/src/db/schema.ts

```diff
@@ -322,6 +322,7 @@ export const itemPacks = sqliteTable(
   {
     itemId: text('item_id').references(() => item.id, { onDelete: 'cascade' }),
     packId: text('pack_id').references(() => pack.id, { onDelete: 'cascade' }),
+    quantity: integer('quantity').notNull().default(1),
```

Comment:
Please remove the default value. It is preferable to make it nullable then fill existing item_packs with quantity retrieved from item with id itemId than assuming the default quantity is one.
@taronaleksanian What do you think about this?

Reply:
to fill existing items we need to write a script that will handle it, which will update corresponding items, so I'm ok with this @Tadjaur